src/modules/song/api.ts

```diff
@@ -45,7 +49,7 @@ export const extendedApi = api.injectEndpoints({
       async onQueryStarted(body, { dispatch, queryFulfilled }) {
         try {
           await queryFulfilled;
-        } catch ({ error }) {
+        } catch (err) {
```

Comment:
can we do `error` vs `err`. I try to be explicit when possible. i've seen places where e is used for event and error and it causes confusion as to what it is. In this case err cannot be confused with event lol but still 🤷🏽‍♂️